templates/kmmo.yaml

```diff
@@ -442,16 +442,16 @@ spec:
         - --config=controller_config.yaml
         env:
         - name: RELATED_IMAGE_WORKER
-          value: gcr.io/k8s-staging-kmm/kernel-module-management-worker:latest
+          value: gcr.io/k8s-staging-kmm/kernel-module-management-worker:v20240306-v2.0.2
         - name: OPERATOR_NAMESPACE
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
         - name: RELATED_IMAGE_BUILD
           value: gcr.io/kaniko-project/executor:latest
         - name: RELATED_IMAGE_SIGN
-          value: gcr.io/k8s-staging-kmm/kernel-module-management-signimage:v20240131-v2.0.1
-        image: gcr.io/k8s-staging-kmm/kernel-module-management-operator:v20240131-v2.0.1
+          value: gcr.io/k8s-staging-kmm/kernel-module-management-signimage:v20240306-v2.0.2
+        image: gcr.io/k8s-staging-kmm/kernel-module-management-operator:v20240306-v2.0.2
```

Comment:
Q1 releases?  Aren't there a newer ones?

Reply:
There is a June 2.1.1 release. That didn't work right out of the box so I updated to the most recent that works.

I will update to the 2.1.1 later when I have more time to look at the issues.